python/whylogs/core/constraints/metric_constraints.py

```diff
@@ -61,15 +77,23 @@ def _validate_metrics(self, metrics: List[Metric]) -> bool:
                 return False
         return True
 
-    def validate(self, dataset_profile: DatasetProfileView) -> bool:
```

Comment:
similarly I think we should have the same return type on the original` validate()` method, and add a new method that returns the Tuple

Reply:
done!